front/app/containers/ProjectsShowPage/shared/header/ProjectHeader.tsx

```diff
@@ -37,6 +38,15 @@ const Container = styled.div`
   `}
 `;
 
+const TopBar = styled.div`
+  display: flex;
+  justify-content: space-between;
+
```

Comment:
I'd add a bit of bottom margin (e.g. 20px) to make the top bar it stand out a bit more when there's a header image underneath it (now it kind of sticks to the header)

Reply:
Nitpicking, but I'd add `align-items: center` here as well just to make sure the left-hand and right-hand buttons are centered

Reply:
Good point, will add a bit of margin! As for the `align-items: center` part: again same question, should I also update it in [IdeasShow](https://github.com/CitizenLabDotCo/citizenlab/blob/master/front/app/containers/IdeasShow/index.tsx#L157)?

Reply:
Yes, feel free to change it there as well